models/electric_relaxation/src/utils/set_paths.py

```diff
@@ -14,7 +14,7 @@ def get_artifacts_path(partition_name):
 
 def get_data_path(data_name):
     '''
-    E.g., The data is saved in data/raw/raw.parquet
+    E.g., The data is saved in data/raw/raw_calibration.parquet
```

Comment:
I'm not sure we need to denote it "raw_" when it is in the raw folder, but we haven't agreed on any conventions yet. But I would probably go with just "calibration.parquet"

Reply:
This is Sara's model. I haven't updated it yet.

Reply:
Noted